lib/src/__tests__/DateTimePickerTestingLib.test.tsx

```diff
@@ -96,4 +96,20 @@ describe('<DateTimePicker />', () => {
     fireEvent.click(screen.getByLabelText('open next view'));
     expect(screen.getByLabelText('open next view')).toBeDisabled();
   });
+
+  it('allows to select the same day and move to the next view', () => {
+    const onChangeMock = jest.fn()
```

Comment:
Interesting, I didn't realize yet that the CI isn't testing that prettier runs.
Do you want to copy & paste the prettier configuration of the main repository in a follow-up pull request?

```suggestion
    const onChangeMock = jest.fn();
```

Reply:
![image](https://user-images.githubusercontent.com/16926049/88181092-03f6e080-cc37-11ea-9144-f47b10bdc112.png)
It is checking

Reply:
Haha it stopped working after adding command `yarn prettier` :D

Reply:
https://github.com/mui-org/material-ui-pickers/pull/2023